timeseries/tests/unittests/test_predictor.py

```diff
@@ -179,8 +179,7 @@ def test_given_hyperparameters_when_predictor_called_and_loaded_back_then_all_mo
 @pytest.mark.parametrize(
     "hyperparameters",
     [
-        {"ETS": {"maxiter": 1}, "SimpleFeedForward": {"epochs": 1}},
-        {"ETS": {"maxiter": 1}, "SimpleFeedForward": {"epochs": ag.Int(1, 3)}},
+        {"ETS": {"maxiter": ag.space.Categorical(1)}, "SimpleFeedForward": {"epochs": ag.space.Int(1, 3)}},
```

Comment:
This test used to rely on the undocumented behavior where the HPs will be overloaded based on the `default_hpo` dict (i.e., in this a search space would be constructed for ETS).

We can either update the test (as done here), or change the logic such that some models can be trained with HPO while other models are trained with fixed configs ([as described above](https://github.com/awslabs/autogluon/pull/2189/files#r989051212))

Reply:
If you think we can have such mixed configs we should add test cases for those as well..